[2024-05-22T01:26:54.732+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-22T01:26:54.796+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: kaggle_dataset.transformacion manual__2024-05-22T01:16:10.864259+00:00 [queued]>
[2024-05-22T01:26:54.823+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: kaggle_dataset.transformacion manual__2024-05-22T01:16:10.864259+00:00 [queued]>
[2024-05-22T01:26:54.824+0000] {taskinstance.py:2306} INFO - Starting attempt 3 of 7
[2024-05-22T01:26:55.154+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): transformacion> on 2024-05-22 01:16:10.864259+00:00
[2024-05-22T01:26:55.169+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=504) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-05-22T01:26:55.172+0000] {standard_task_runner.py:63} INFO - Started process 528 to run task
[2024-05-22T01:26:55.172+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'kaggle_dataset', 'transformacion', 'manual__2024-05-22T01:16:10.864259+00:00', '--job-id', '10', '--raw', '--subdir', 'DAGS_FOLDER/kaggle_dataset.py', '--cfg-path', '/tmp/tmpwotjc22o']
[2024-05-22T01:26:55.175+0000] {standard_task_runner.py:91} INFO - Job 10: Subtask transformacion
[2024-05-22T01:26:55.296+0000] {task_command.py:426} INFO - Running <TaskInstance: kaggle_dataset.transformacion manual__2024-05-22T01:16:10.864259+00:00 [running]> on host b17c52120858
[2024-05-22T01:26:55.589+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='kaggle_dataset' AIRFLOW_CTX_TASK_ID='transformacion' AIRFLOW_CTX_EXECUTION_DATE='2024-05-22T01:16:10.864259+00:00' AIRFLOW_CTX_TRY_NUMBER='3' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-05-22T01:16:10.864259+00:00'
[2024-05-22T01:26:55.592+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-22T01:26:55.619+0000] {logging_mixin.py:188} INFO - Hola Mundo
[2024-05-22T01:26:55.620+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-05-22T01:26:55.621+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-22T01:26:55.641+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=kaggle_dataset, task_id=transformacion, run_id=manual__2024-05-22T01:16:10.864259+00:00, execution_date=20240522T011610, start_date=20240522T012654, end_date=20240522T012655
[2024-05-22T01:26:55.714+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-22T01:26:55.851+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-05-22T01:26:55.858+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
